packages/language-server/package.json

```diff
@@ -24,6 +24,7 @@
     "node": ">=10.0.0"
   },
   "dependencies": {
+    "@vscode-logging/types": "0.1.3",
```

Comment:
imports in `api.d.ts` need to appear in deps instead of dev-deps to enable end user consumption